src/pages/Route/typing.d.ts

```diff
@@ -113,6 +116,7 @@ declare namespace RouteModule {
   // Request Body or Response Data for API
   type Body = {
     id?: number;
+    status: boolean;
```

Comment:
do we need to add a boolean flag in route? 

https://github.com/apache/apisix/blob/master/doc/admin-api.md#route

we'll remove mysql soon.

Reply:
If moved `status` to abisix, it could be a good extension as a basis for route lifecycle management, and if not, `status` only a a new field of the route table in the current mysql, which is not associated with other tables, it would be easier to migrate in new manager api  .